blobfuse/directoryapis.cpp

```diff
@@ -228,13 +228,26 @@ int azs_rmdir(const char *path)
 int azs_statfs(const char *path, struct statvfs *stbuf)
 {
     std::string pathString(path);
-    const char * mntPath;
-    std::string mntPathString = prepend_mnt_path_string(pathString);
-    mntPath = mntPathString.c_str();
 
-    int res = statvfs(mntPath, stbuf);
-    if (res == -1)
-        return -errno;
+    int dir_result = is_directory_empty(str_options.containerName, "/", pathString.substr(1));
+    bool path_exists = false;
+    if(strlen(path) == 1 || dir_result == D_NOTEMPTY)
```

Comment:
what does it mean if path length is 1? why we consider it to exist?

Reply:
It is root of fuse "/"

Reply:
why not to check if it's "/" explicitly?

Reply:
I was following the precedence in the code to be consistent. It seems Adam used strlen here as well: https://github.com/Azure/azure-storage-fuse/blob/ed79ab5108cbcb70dadc540e435d5622938c3d58/blobfuse/utilities.cpp#L340

Reply:
@asorrin-msft keep this or change to path=="/" ?